model/Security/Classes/CvssV4VulnAssessmentRelationship.md

```diff
@@ -0,0 +1,81 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CvssV4VulnAssessmentRelationship
+
+## Summary
+
+Provides a CVSS version 4.0 assessment for a vulnerability.
+
+## Description
+
+A CvssV4VulnAssessmentRelationship relationship describes the determined score, severity, and vector of a vulnerability using version 4.0 of the Common Vulnerability Scoring System (CVSS) as defined on [https://www.first.org/cvss/v4.0/specification-document](https://www.first.org/cvss/v4.0/specification-document). It is intented to communicate the results of using a CVSS calculator.
+
+**Constraints**
+
+- The value of severity must be one of 'none', 'low', 'medium', 'high' or 'critical'.
+- Absence of the property shall be interpreted as 'none'.
+- The relationship type must be set to hasAssessmentFor.
+
+**Syntax**
+
+```json
+{
+  "@type": "CvssV4VulnAssessmentRelationship",
+  "@id": "urn:spdx.dev:cvssv4-cve-2020-28498",
+  "relationshipType": "hasAssessmentFor",
+  "severity": "medium",
+  "score": 6.8,
+  "vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:N/A:N",
```

Comment:
From a quick glance, should this start with "CVSS:4"?

Reply:
Addressed in latest commit, thanks!